pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -1312,3 +1378,136 @@ func updatePipelineRunStatusFromRuns(logger *zap.SugaredLogger, pr *v1beta1.Pipe
 		}
 	}
 }
+
+func updatePipelineRunStatusFromChildRefs(logger *zap.SugaredLogger, pr *v1beta1.PipelineRun, trs []*v1beta1.TaskRun, runs []*v1alpha1.Run) map[string]string {
```

Comment:
I don't think this function needs to return anything

Reply:
See above at https://github.com/tektoncd/pipeline/pull/4739#discussion_r844421328 =)